files/build_templates/sonic_debian_extension.j2

```diff
@@ -89,6 +89,12 @@ sudo cp {{platform_common_py2_wheel_path}} $FILESYSTEM_ROOT/$PLATFORM_COMMON_PY2
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install $PLATFORM_COMMON_PY2_WHEEL_NAME
 sudo rm -rf $FILESYSTEM_ROOT/$PLATFORM_COMMON_PY2_WHEEL_NAME
 
+# Install built Python Click package (and its dependencies via 'apt-get -y install -f')
+# Do this before installing sonic-utilities so that it doesn't attempt to install
+# an older version as part of its dependencies
+sudo dpkg --root=$FILESYSTEM_ROOT -i target/debs/python-click*_all.deb || \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
+
```

Comment:
Prefer
```
pip install click
```
if possible

Reply:
Possible, but because sonic-utilities is built as a .deb package, it attempts to install its dependencies as .deb packages, also. We've encountered this in the past. If we install via pip, we will have two versions of Click installed on the system. The default Python paths should look in the pip installation location (/usr/local/lib/python2.7/dist-packages) before the apt installation location (/usr/lib/python2.7/dist-packages), so in theory it would work, but it's messy and provides the potential for the wrong version to be used. This solution ensures only the 6.7 version of Click is installed on the system.